packages/zengl/test_zengl.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+
+
+@pytest.mark.skip_refcount_check
+@pytest.mark.skip_pyproxy_check
+def test_render_with_webgl2(selenium_standalone):
```

Comment:
Try using `@run_in_pyodide(packages=["zengl"])` and `@pytest.mark.xfail_browsers`